### PR TITLE
Update naming to use pgAdmin CR instance UID

### PR DESCRIPTION
**Pull request by tjmoore4**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [x] New feature
 - [ ] Bug fix
 - [ ] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

Update all relevant names and impacted tests.

**Other Information**:
Issue: PGO-591

### CODE REVIEW

File: internal/controller/standalone_pgadmin/pod_test.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ containers:
     valueFrom:
       secretKeyRef:
         key: password
-        name: pgadmin-standalone-pgadmin
+        name: pgadmin-
```
Comment (jmckulk) [inline internal/controller/standalone_pgadmin/pod_test.go:73]: I assume since the `cr` doesn't really exist, the UID is just an empty string?

**Review by jmckulk — APPROVED** [review]
lgtm

**Review by benjaminjb — APPROVED** [review]
I assume the KUTTL test is working locally for you now?

**Comment by tjmoore4** [comment]
> I assume the KUTTL test is working locally for you now?

Yep, working as expected.

File: internal/controller/standalone_pgadmin/pod_test.go (hunk 1 above)
Comment (tjmoore4) [reply internal/controller/standalone_pgadmin/pod_test.go:73]: Yes, that's right. I think that's ok for these tests.

**Pull request merged**